app/src/main/java/uk/gov/onelogin/features/domain/SetFeatureFlags.kt

```diff
@@ -0,0 +1,47 @@
+package uk.gov.onelogin.features.domain
+
+import android.util.Log
+import javax.inject.Inject
+import uk.gov.android.features.FeatureFlags
+import uk.gov.android.features.InMemoryFeatureFlags
+import uk.gov.onelogin.appinfo.apicall.domain.model.AppInfoData
+import uk.gov.onelogin.appinfo.source.domain.model.AppInfoLocalState
+import uk.gov.onelogin.appinfo.source.domain.source.AppInfoLocalSource
+import uk.gov.onelogin.features.AppCheckFeatureFlag
+
+fun interface SetFeatureFlags {
```

Comment:
Verbs have no place in Type names (Classes and Interfaces). This can be as simple as "er"ing the verb you wish to use, e.g. FeatureFlagSetter

Reply:
Agreed, habits die hard 🤦